app/views/layouts/_sidebar_tags.html.erb

```diff
@@ -1,14 +1,14 @@
 <nav class="mb-6" aria-label="Secondary sidebar nav">
   <% if user_signed_in? %>
     <header class="p-2 pr-0 flex items-center justify-between">
-      <h3 class="crayons-subtitle-3">My Tags</h3>
+      <h3 class="crayons-subtitle-3"><%= t("core.my_tags") %></h3>
       <a href="/dashboard/following_tags" class="crayons-btn crayons-btn--icon crayons-btn--ghost-dimmed" aria-label="Customize tag priority" title="Customize tag priority">
         <%= inline_svg_tag("cog.svg", class: "crayons-icon") %>
       </a>
     </header>
     <div id="sidebar-nav-followed-tags" class="overflow-y-auto mb-2 sidebar-nav-followed-tags" style="max-height: 42vh;"></div>
   <% else %>
-    <h3 class="crayons-subtitle-3 p-2">Popular Tags</h3>
+    <h3 class="crayons-subtitle-3 p-2"><%= t("popular_tags") %></h3>
```

Comment:
This key doesn't seem to exist at all. But even if it did it should probably be in the `core` namespace:

```suggestion
    <h3 class="crayons-subtitle-3 p-2"><%= t("core.popular_tags") %></h3>
```